pilot/pkg/model/sidecar.go

```diff
@@ -31,6 +32,12 @@ const (
 	wildcardService   = host.Name("*")
 )
 
+var sidecarScopeKnownConfigTypes = map[resource.GroupVersionKind]struct{}{
+	ServiceEntryKind:    {},
+	VirtualServiceKind:  {},
+	DestinationRuleKind: {},
+}
```

Comment:
Why not include sidecarScope itself

Reply:
This value has nothing to do with a sidecarScope instance.